perception/traffic_light_visualization/src/traffic_light_roi_visualizer/nodelet.cpp

```diff
@@ -95,27 +97,22 @@ bool TrafficLightRoiVisualizerNodelet::createRect(
   if (result.label.find("red") != std::string::npos) {
     color = cv::Scalar{255, 0, 0};
   } else if (result.label.find("yellow") != std::string::npos) {
-    color = cv::Scalar{0, 255, 0};
+    color = cv::Scalar{255, 255, 0};
   } else if (result.label.find("green") != std::string::npos) {
-    color = cv::Scalar{0, 0, 255};
+    color = cv::Scalar{0, 255, 0};
   } else {
     color = cv::Scalar{255, 255, 255};
   }
```

Comment:
Could you change these colors with:
- `#FE9595`
- `#FEFA95`
- `#95FEA1`
- `#FAFAFA`

Which are hopefully in order (from chatgpt):
```
  {254, 149, 149},
  {254, 250, 149},
  {149, 254, 161},
  {250, 250, 250}
```

Otherwise it won't pass the contrast readability check: https://webaim.org/resources/contrastchecker/